src/routes/write-exercise/__tests__/WriteExerciseScreen.spec.tsx

```diff
@@ -28,9 +28,15 @@ jest.mock('react-native/Libraries/LogBox/Data/LogBoxData')
 
 describe('WriteExerciseScreen', () => {
   beforeEach(() => {
+    // mocking math makes normally random order of documents predictable
+    jest.spyOn(Math, 'random').mockReturnValue(0.9)
     jest.clearAllMocks()
   })
 
+  afterEach(() => {
+    jest.spyOn(Math, 'random').mockRestore()
```

Comment:
```suggestion
    jest.restoreAllMocks()
```
not tested but should do the same